docs/proposals/ldap-group-sync.md

```diff
@@ -0,0 +1,112 @@
+# Syncing LDAP Groups to OpenShift Groups
+
+OpenShift cluster admins need a way to sync their OpenShift `Groups` to external records in an LDAP server. This requires that we have a way of retrieving the LDAP user entries that are the members of a group, determining the OpenShift `Group` name for this collection, and updating the OpenShift record, removing the record as necessary when the corresponding LDAP record is removed. Necessary information about the LDAP server set-up and schema will be provided by the user in a `yaml` configuration file and pointed to in the `oadm sync-groups` command.
```

Comment:
"retrieving the LDAP users" (not entries)

"configuration file and referenced by the `oadm sync-groups` command."